service_status/lambda_handler.py

```diff
@@ -0,0 +1,72 @@
+import json
+import re
+import traceback
+
+from common.constant import NETWORKS
+from common.repository import Repository
+from common.utils import Utils
+from service_status.service_status import ServiceStatus
+
+NETWORKS_NAME = dict((NETWORKS[netId]['name'], netId) for netId in NETWORKS.keys())
+obj_util = Utils()
+db = dict((netId, Repository(net_id=netId)) for netId in NETWORKS.keys())
+route_path = {}
+
+
+def request_handler(event, context):
+    print(event)
+    if 'path' not in event:
+        print("request_handler::path: ", None)
+        return get_response("400", "Bad Request")
+    try:
+        payload_dict = None
+        path = event['path'].lower()
```

Comment:
Lets not use httpMethod in the event as we now have a standalone lambda. We can modify the cloudwatch rule to pass in the required params only

Reply:
resolved